### ISSUE TITLE
No open source license for this repository

### ISSUE THREAD

**Issue by olivia-livs** [OpenSC4-org/OpenSC4#5]
For the project to be truly open source it needs to have an open source license. Without a license it's not clear if anyone is allowed to use, modify, or share this repository. You can choose whatever license you want :) I'll show some resources that I personally found helpful.
https://choosealicense.com/
https://opensource.org/licenses
If you have included a license already and I just haven't found it let me know haha.

**Comment by TorchedSammy** [comment]
ive actually opened an issue about this on the *original* repo at https://github.com/AJEKsoft/OpenSC4/issues/5 (funnily enough same issue number) and im not sure how to go about adding a license here since the code is really theirs without a license.

**Comment by olivia-livs** [comment]
also could i ask? why is this repository completely separate and not a fork of https://github.com/AJEKsoft/OpenSC4/? They seem to have the same git history

**Comment by olivia-livs** [comment]
the original developers have been inactive for months :sob:

**Comment by TorchedSammy** [comment]
> the original developers have been inactive for months 😭

yes, that's why. me and a few other people are starting it up again, and having this org repo allows for one central repo for contributions and other things

**Comment by AdrienJaguenet** [comment]
I'd rather use (A)GPLv3 but I'm willing to use another license if other contributors prefer.

**Comment by TorchedSammy** [comment]
I don't think the additional clause in the agpl is needed, so just plain gpl works fine.

**Comment by AdrienJaguenet** [comment]
I'm thinking AGPL because networking features are planned

**Comment by olivia-livs** [comment]
I prefer GPL personally because im not a big fan of someone potentially using open source code in proprietary software. AGPL is fine with me :D!

**Comment by stephen-zhao** [comment]
Ya that's my main concern with MIT too
